tests/src/common/Wsk.scala

```diff
@@ -149,9 +149,14 @@ trait ListOrGetFromCollection extends FullyQualifiedNames {
      */
     def get(
         name: String,
+        summary: Boolean = false,
         expectedExitCode: Int = SUCCESS_EXIT)(
             implicit wp: WskProps): RunResult = {
-        cli(wp.overrides ++ Seq(noun, "get", "--auth", wp.authKey, fqn(name)), expectedExitCode)
+        var cliparms = wp.overrides ++ Seq(noun, "get", "--auth", wp.authKey, fqn(name))
+        if (summary) {
+            cliparms = cliparms ++ Seq("--summary")
+        }
+        cli(cliparms, expectedExitCode)
```

Comment:
Nice example.  Scala never ceases to amaze me :-)  Personally, I like the readability of the existing test code.  Unless strong objections, I'll keep it as-is.

I made changes to the namespace matching code per comment from @rabbah .  Rebased and ready for merging.

PG 142